metsim/constants.py

```diff
@@ -0,0 +1,56 @@
+"""
+Stores default constants, parameters, and options.  
+"""
```

Comment:
Would be good to organize the file so that physical constants (value is always the same), conversion parameters (`SECPERRAD`) parameters (tunable) and options are organized in blocks with clear identification what is what.